Code Samples/Fib/.vscode/tasks.json

```diff
@@ -4,7 +4,7 @@
     "version": "2.0.0",
```

Comment:
did you remove the colon? I Don't know what this diff means>?

Reply:
No, there's no colon removed. Are you seeing a change on that line?

Reply:
the colon is marked as "red" I don't know why

Reply:
Yeah, I think that's just some GitHub visualization bug -- it's red in the pre-post versions. The raw data doesn't have that.

Reply:
It's red because the comments as messing up the formatter since comments are not valid in JSON.

Reply:
Ah, that makes sense -- VS Code uses a json parser for tasks.json that allows comments though, so it's just a GitHub issue.